templates/joplin/meta.yaml

```diff
@@ -0,0 +1,48 @@
+name: Joplin
+description: Joplin is an open source note-taking app. Capture your thoughts and securely access them from any device.
+instructions: null
+changeLog:
+  - date: 2023-03-16
+    description: First Release
+links:
+  - label: Website
+    url: https://joplinapp.org/
+  - label: Github
+    url: https://github.com/laurent22/joplin/
+  - label: Docker
+    url: https://hub.docker.com/r/joplin/server
+contributors:
+  - name: BerkSMBL
+    url: https://berksmbl.com
+schema:
+  type: object
+  required:
+    - projectName
+    - domain
+    - secure
+    - appServiceName
+    - appServiceImage
+    - databaseServiceName
+  properties:
+    projectName:
+      type: string
+      title: Project Name
+    domain:
+      type: string
+      title: Domain
+    secure:
```

Comment:
let's not expose this config, and make it secure by default

Reply:
I added this option later because: If I do not specify https or http to the domain variable in the APP ENV, joplin does not allow connection.

If I specify https, it does not accept incoming connections over http.

Reply:
Shouldn't be a problem since Easypanel is requesting a certificate etc.
If people want http they're still able to change it afterwards.

Reply:
![image](https://user-images.githubusercontent.com/10000339/225606197-b2e39f13-a85b-49a5-ae75-ab667ebe8556.png)

Reply:
I have presented this option considering the possibility that users will not use SSL in the domain they have specified since they will not know the default domain name by heart.

Reply:
> it does not accept incoming connections over http

I think Easypanel would redirect that to https automatically. Right?